power_juices/mcpeachpies_power_juices/data/mcpeachpies_power_juices/loot_tables/recipes/apple_juice.json

```diff
@@ -0,0 +1,34 @@
+{
+  "type": "minecraft:generic",
+  "pools": [
+    {
+      "rolls": 4,
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:air"
+        }
+      ]
+    },
+    {
+      "rolls": 1,
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:potion",
+          "functions": [
+            {
+              "function": "minecraft:set_name",
+              "entity": "this",
+              "name": "Apple Juice"
```

Comment:
This should be replaced with `{ "text": "Apple Juice", "italic": false }` (and the same for others) to get rid of the italics in the name (not the same as the original Power Juices but imo looks better)

Reply:
yeah this comment will have to be good enough, can't figure out how requesting changes works 🥴

Reply:
If that's desired, sure.  I can change it tomorrow

Reply:
I'll just merge this PR and do it myself actually, it's a simple change, no need to agonize over the process I realize